obspy/taup/tests/test_velocity_model.py

```diff
@@ -20,14 +20,17 @@
 
 class TauPyVelocityModelTestCase(unittest.TestCase):
     def test_read_velocity_model(self):
-        for i in range(0, 2):
+        for i in range(0, 4):
```

Comment:
I don't know why this loop was written this way in the first place, but now that it's 4 elements long, it makes more sense to be `for filename in ['iasp91.tvel', 'iasp91_w_comment.tvel', ...]:` and leave out `i` entirely.